app/src/main/java/com/framgia/gotosalon/data/source/SavingAccountPreference.java

```diff
@@ -0,0 +1,47 @@
+package com.framgia.gotosalon.data.source;
+
+import android.content.SharedPreferences;
+
+import com.framgia.gotosalon.data.model.Account;
+
+public class SavingAccountPreference {
+    private static final String KEY_PREFERENCE_EMAIL = "DOHAIHA";
```

Comment:
em có biết biến này dùng để làm gì và được lưu ở đâu ko

Reply:
đó là key để truyền vào methood dạng key-value và cặp key-value này đc lưu ở  DDMS/File Exploer/data/data/"package name"/shared_prefers dưới dạng .xml ạ

Reply:
oh vậy thì a suggest e đặt name của nó là KEY_PREFERENCE_EMAIL
\
```suggestion
    private static final String KEY_PREFERENCE_EMAIL = "KEY_PREFERENCE_EMAIL";
```
tương tự cho các biến khác